source/configuration_manual/config_file/index.rst

```diff
@@ -1,11 +1,294 @@
-==================
-Configuration file
-==================
+.. _config_file_syntax:
 
-.. toctree::
-   :maxdepth: 1
-   :glob:
+==============================
```

Comment:
This should be same length as title.

Reply:
This was in the "move" commit? I just moved the stuff without changing. But I suppose might as well fix the title lengths while moving.